main/features.py

```diff
@@ -1,39 +1,42 @@
 """MITxOnline feature flags"""
-import os
 from functools import wraps
 
 from django.conf import settings
-from posthog import Posthog
-
-if "IN_TEST_SUITE" not in os.environ:
-    posthog = Posthog(settings.POSTHOG_API_TOKEN, host=settings.POSTHOG_API_HOST)
-else:
-    posthog = None
 
 
 IGNORE_EDX_FAILURES = "IGNORE_EDX_FAILURES"
 SYNC_ON_DASHBOARD_LOAD = "SYNC_ON_DASHBOARD_LOAD"
 ENABLE_ADDL_PROFILE_FIELDS = "ENABLE_ADDL_PROFILE_FIELDS"
 ENABLE_NEW_DESIGN = "mitxonline-new-product-page"
+ENABLE_NEW_HOME_PAGE_FEATURED = "mitxonline-new-featured-carousel"
+ENABLE_NEW_HOME_PAGE_HERO = "mitxonline-new-featured-hero"
+ENABLE_NEW_HOME_PAGE_VIDEO = "mitxonline-new-home-page-video-component"
 
 
-def is_enabled(name, default=None):
+def is_enabled(name, default=None, unique_id=settings.HOSTNAME):
     """
     Returns True if the feature flag is enabled
 
     Args:
         name (str): feature flag name
         default (bool): default value if not set in settings
+        unique_id (str): person identifier passed back to posthog which is the display value for person. I recommend
+                         this be a readable id for logged-in users to allow for user flags as well as troubleshooting.
```

Comment:
I'd explicitly say User.id for logged in users.

Reply:
gotcha